test/e2e/util/kubectl.go

```diff
@@ -104,6 +116,43 @@ func KubectlDescribe(kubeconfigPath, podName, namespace string) (string, error)
 	return Kubectl(args)
 }
 
+// KubectlDescribe executes "kubectl describe" given a list of arguments.
+func KubectlCurlPod(kubeconfigPath string) (string, error) {
```

Comment:
You need to create this pod in the same namespace as the otel-collector. Have this function accept the namespace as an argument